opencga-core/src/main/java/org/opencb/opencga/core/models/AclEntryList.java

```diff
@@ -0,0 +1,6 @@
+package org.opencb.opencga.core.models;
+
+import java.util.ArrayList;
+
+public class AclEntryList<E extends Enum<E>> extends ArrayList<AclEntry<E>> {
```

Comment:
Hummm.. not sure about this thing...
This kind of obscure "aliasing" might have weird implications. It looks like and object, but it's an array.
With this, the REST will return a double array, something like this:
```json
{
  "responses": [
    {
       "resultType" : "AclEntryList",
       "numResults" : 1
       "results": [ [ {"member": "m1" }, {"member": "m2" }, {"member": "m3" } ] ]
    }
  ]
}
```

Reply:
Yes, and that's exactly what we want. The array of results will contain as many AclEntryLists as entries have been requested and the AclEntryLists as many elements as members have been requested